app/models/etsource/base.rb

```diff
@@ -116,6 +116,12 @@ def get_latest_import_sha
       nil
     end
 
+    def last_updated_at(folder)
+      unless Rails.env.test?
+        git.log(1).object(folder).map(&:date).first
```

Comment:
Thanks for your comment! I did know of the import commits functionality but didn't know the technical details :smile: